### PR TITLE
Update tests urls

### PR DESCRIPTION
**Pull request by drEfrati**
1. Update kin-sdk-android to latest version
2. Change tests urls from http to https
whitelist service tests will be updated in a later version(soon)

### CODE REVIEW

File: migration/src/androidTest/kotlin/kin/sdk/migration/MigrationManagerIntegrationTest.kt [hunk 1]
```diff
@@ -26,19 +26,21 @@ import kotlin.test.fail
 @Suppress("FunctionName")
 class MigrationManagerIntegrationTest {
 
-    private val sdkTestNetworkUrl = "http://horizon-testnet.kininfrastructure.com/"
+    private val sdkTestNetworkUrl = "https://horizon-testnet.kininfrastructure.com/"
     private val sdkTestNetworkId = "Kin Testnet ; December 2018"
     private val coreTestNetworkUrl = "https://horizon-playground.kininfrastructure.com/"
     private val coreTestNetworkId = "Kin Playground Network ; June 2018"
     private val coreIssuer = "GBC3SG6NGTSZ2OMH3FFGB7UVRQWILW367U4GSOOF4TFSZONV42UJXUH7"
+    private val migrateServiceUrl = "https://migration-devplatform-production.developers.kinecosystem.com/migrate?address="
```
Comment (yosriz) [inline migration/src/androidTest/kotlin/kin/sdk/migration/MigrationManagerIntegrationTest.kt]: why production url?

File: sample/src/main/java/kin/sdk/migration/sample/OnBoarding.java [hunk 2]
```diff
@@ -20,11 +18,13 @@
...
-    private static final String TEST_CORE_URL_CREATE_ACCOUNT = "http://friendbot-playground.kininfrastructure.com/?addr=";
-    private static final String TEST_SDK_URL_FUND = "http://friendbot-testnet.kininfrastructure.com/fund?addr=%s&amount=" + String.valueOf(FUND_KIN_AMOUNT);
+    private static final String TEST_SDK_URL_CREATE_ACCOUNT =
+        "https://friendbot-testnet.kininfrastructure.com?addr=%s&amount=" + String.valueOf(FUND_KIN_AMOUNT);
+    private static final String TEST_CORE_URL_CREATE_ACCOUNT = "https://friendbot-playground.kininfrastructure.com/?addr=";
+    private static final String TEST_SDK_URL_FUND =
+        "https://friendbot-testnet.kininfrastructure.com/fund?addr=%s&amount=" + String.valueOf(FUND_KIN_AMOUNT);
     private static final String TEST_CORE_URL_FUND =
-        "http://faucet-playground.kininfrastructure.com/fund?account=%s&amount=" + String.valueOf(FUND_KIN_AMOUNT);
+        "https://faucet-playground.kininfrastructure.com/fund?account=%s&amount=" + String.valueOf(FUND_KIN_AMOUNT);
```
Comment (yosriz) [inline sample/src/main/java/kin/sdk/migration/sample/OnBoarding.java]: same as in `kin-core`, there's no https for playground faucet

File: migration/src/androidTest/kotlin/kin/sdk/migration/MigrationManagerIntegrationTest.kt (hunk 1 above)
Comment (drEfrati) [reply migration/src/androidTest/kotlin/kin/sdk/migration/MigrationManagerIntegrationTest.kt]: mistake but anyway all the integration tests are been developed in a different branch and in that branch everything is up to date

File: sample/src/main/java/kin/sdk/migration/sample/OnBoarding.java (hunk 2 above)
Comment (drEfrati) [reply sample/src/main/java/kin/sdk/migration/sample/OnBoarding.java]: Will fix

**Review by yosriz — APPROVED** [review]

**Pull request merged**